src/coalesce-vue-vuetify3/src/components/admin/c-admin-table-toolbar.vue

```diff
@@ -56,27 +51,35 @@
     <v-spacer></v-spacer>
 
     <c-list-page class="c-admin-table-toolbar--page" :list="list" />
+  </v-toolbar>
 
-    <template v-slot:extension>
-      <v-text-field
-        class="c-admin-table-toolbar--search"
-        hide-details
-        prepend-inner-icon="fa fa-search"
-        label="Search"
-        v-model="list.$params.search"
-        density="comfortable"
-        single-line
-        clearable
-      ></v-text-field>
-
-      <v-divider class="hidden-xs-only mx-4 my-0" vertical></v-divider>
-
-      <c-list-filters :list="list" />
+  <v-sheet :color="color">
```

Comment:
I'm not sure about putting this in the toolbar - especially with the amount of dismantling of the actual toolbar structure that it required.

I think it should go in the body of the card, at the top:
![image](https://github.com/user-attachments/assets/c604b422-9d28-47b7-b851-7d824ac349fe)
![image](https://github.com/user-attachments/assets/9c6f1c65-cc39-4bbc-89a3-4c96d63ef3cf)

``` scss
.c-admin-table--description {
  font-size: 12px;
  margin: 4px 16px;
  opacity: 0.9;
}
```

Likewise for the editor

Reply:
That sounds good, I wasn't really sure about where to put this. I started with it in the body, but it looked a bit weird (my font was default size, though). I think the smaller text helps.